config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path.yaml

```diff
@@ -18,7 +18,28 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-        # Copied from config/jobs/kubernetes/sig-testing/verify.yaml.
+        # kind needs /lib/modules and cgroups from the host
+        volumeMounts:
+        - mountPath: /lib/modules
+          name: modules
+          readOnly: true
+        - mountPath: /sys/fs/cgroup
+          name: cgroup
+        # Resource specification copied from config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml.
+        # This job might be similar (to be checked).
         resources:
           requests:
-            cpu: 4
+            # these are both a bit below peak usage during build
+            # this is mostly for building kubernetes
+            memory: "9000Mi"
```

Comment:
pretty sure~2000Mi and 2000m should be plenty, these values are due to building kubernetes which is pretty heavy.